app/controllers/authenticate_controller.rb

```diff
@@ -100,8 +120,24 @@ def k8s_inject_client_cert
 
   private
 
+  def handle_login_error(err)
+    logger.warn("Authentication Error: #{err.inspect}")
```

Comment:
please change this to "Login Error" so we can differentiate it from the "Authentication Error" below

Reply:
also, did we decide this should be a warn and not a debug? 
Are we sure the `err` doesn't contain any sensitive information? 
@shaharglazner do you see any concern here?

Reply:
These are the errors it can produce:

```
    rescue Errors::Authentication::InvalidCredentials
      raise ApplicationController::Unauthorized, "Invalid username or password"
    rescue Errors::Authentication::InvalidOrigin
      raise ApplicationController::Forbidden, "User is not authorized to login from the current origin"
    rescue Errors::Authentication::Security::UserNotAuthorizedInConjur
      raise ApplicationController::Forbidden, "User is not authorized to login to Conjur"
    end
```